Doc/library/stdtypes.rst

```diff
@@ -1376,6 +1376,108 @@ objects that compare equal might have different :attr:`~range.start`,
    single: str (built-in class); (see also string)
    object: string
 
+.. _text-methods-summary:
+
+Text and Binary Sequence Type Methods Summary
+=============================================
+The following table summarizes the text and binary sequence types methods by
+category.
+
+
++--------------------------+------------------------------+------------------------------+
+| Category                 |  str methods                 |  bytes methods               |
++==========================+==============================+==============================+
+| Formatting               |  :meth:`str.format`          |                              |
++                          +------------------------------+------------------------------+
```

Comment:
I think the leftmost ``+`` can/should be a ``|``.

Reply:
Thanks!